<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Amar-PC" name="Test" tests="6" failures="3" timestamp="27 Nov 2015 11:26:34 GMT" time="0.056" errors="0">
  <testcase name="MyTestThree" time="0.011" classname="MavenTestCases.TestClass2"/>
  <testcase name="Test1" time="0.001" classname="MavenTestCases.TestClass1">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;20&amp;gt; but was:&amp;lt;15&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<20> but was:<15>
	at MavenTestCases.TestClass1.Test1(TestClass1.java:13)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.0" classname="MavenTestCases.TestClass1"/>
  <testcase name="Test3" time="0.0" classname="MavenTestCases.TestClass1"/>
  <testcase name="Test4" time="0.0" classname="MavenTestCases.TestClass1">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;20&amp;gt; but was:&amp;lt;15&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<20> but was:<15>
	at MavenTestCases.TestClass1.Test4(TestClass1.java:31)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- Test4 -->
  <testcase name="Test5" time="0.0" classname="MavenTestCases.TestClass1">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;20&amp;gt; but was:&amp;lt;15&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<20> but was:<15>
	at MavenTestCases.TestClass1.Test5(TestClass1.java:37)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- Test5 -->
</testsuite> <!-- Test -->
